net-libs/libwebsockets/libwebsockets-1.7.7.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="canonical libwebsockets.org websocket library"
+HOMEPAGE="https://libwebsockets.org/"
+SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+http2 +server +ssl client ipv6 libev libressl libuv static-libs"
+
+REQUIRED_USE="
+	|| ( client server )
+	client? ( server )
```

Comment:
So, what you're telling me here is that the server portion is required. Always. It isn't actually an option. If that's the case, why do we have a server USE flag?